spanner/value.go

```diff
@@ -3210,6 +3222,12 @@ func decodeProtoEnumArray(pb *proto3.ListValue, t *sppb.Type, rv reflect.Value)
 	// As the ENUM element in the Array is not a pointer type we cannot use decodeValue method
 	// and hence handle it separately.
 	for i, v := range pb.Values {
+		_, isNull := v.Kind.(*proto3.Value_NullValue)
+		// As the ENUM elements in the array are value type and not pointer type,
+		// we cannot support NULL values in the array
+		if isNull {
+			return errNilNotAllowed(ptr, "*[]*protoreflect.Enum")
```

Comment:
The error msg is:

... Use *[]*protoreflect.Enum, an array with pointer type elements to read Null values.

Is this expected?